src/components/menu/templates/main.tsx

```diff
@@ -44,6 +44,7 @@ const items = {
     'menu-theming-css-variables': '/docs/theming/css-variables',
     'Colors': '/docs/theming/colors',
     'Themes': '/docs/theming/themes',
+    'Dark Mode': '/docs/theming/dark-mode',
```

Comment:
Perry's translations project is now live, so this token should change to match the others. here, it'd be "menu-theming-dark-mode" instead of "Dark Mode"

Reply:
Just curious why isn't that on the others?

Reply:
haha not sure. they must have been missed, i guess

Reply:
Spoke with Perry and this has to be added for English only and then the translate service handles the rest, so added it.